register/blake3/multihash_blake3.go

```diff
@@ -18,6 +18,6 @@ import (
 )
 
 func init() {
-	multihash.Register(multihash.BLAKE3, func() hash.Hash { h := blake3.New(32, nil); return h })
+	multihash.Register(multihash.BLAKE3, func() hash.Hash { h := blake3.New(128, nil); return h })
```

Comment:
Shouldn't it call `RegisterVariableSize` ?
```suggestion
	multihash.RegisterVariableSize(multihash.BLAKE3, func(size int) (hash.Hash, bool) {
	  if size == -1 {
	    size = 32
	  } else if size > 128 || size <= 0 {
	    return nil, false
	  }
	  h := blake3.New(size, nil)
	  return h, true
	})
```
I would like for the magic numbers to be unmagified if possible, if it's untidy to do so, just skip it.